tests/unit/utils/test_utils_record.py

```diff
@@ -294,3 +300,9 @@ def test_get_value_returns_none_on_index_error():
     })
 
     assert get_value(single_title, 'titles.title[1]') is None
+
+
+def test_get_record_type_returns_none_when_no_type():
+    record = Record({})
+
+    assert get_record_type(record) is None
```

Comment:
Will document it, sure thing.

Also, I can leave the test with xfail but I think it'll be better to have the function repaired. In order to do that I need a mapping `schemas - pid types` or at least predefined list of schemas (`config.py`?). Any idea where to get it from?